mail_debrand/models/mail_render_mixin.py

```diff
@@ -26,37 +26,28 @@ def remove_href_odoo(
             # We don't want to change what was explicitly added in the message body,
             # so we will only change what is before and after it.
             if to_keep:
-                to_change = value.split(to_keep)
-            else:
-                to_change = [value]
-                to_keep = ""
-            new_parts = []
-            for part in to_change:
-                tree = html.fromstring(part)
-                if tree is None:
-                    new_parts.append(part)
-                    continue
-                odoo_anchors = tree.xpath('//a[contains(@href,"odoo.com")]')
-                for elem in odoo_anchors:
-                    parent = elem.getparent()
-                    previous = elem.getprevious()
-
-                    if remove_before and not remove_parent and previous is not None:
-                        # remove 'using' that is before <a and after </span>
-                        previous.tail = ""
-                    if remove_parent and len(parent.getparent()):
-                        # anchor <a href odoo has a parent powered by that must be removed
+                value = value.replace(to_keep, "<body_msg></body_msg>")
+            tree = html.fromstring(value)
+            odoo_anchors = tree.xpath('//a[contains(@href,"odoo.com")]')
```

Comment:
To solve https://github.com/OCA/social/issues/884 can workaround it using:
`odoo_anchors = tree.xpath('//a[contains(@href,"odoo.com") and not(contains(@href,"odoo.com/page/tour"))]')`
And put a comment why we are adding this "not contains".

It's just adding `and not(contains(@href,"odoo.com/page/tour"))` to exclude the Odoo Tour link.